include/seqan3/io/views/detail/take_until_view.hpp

```diff
@@ -103,8 +103,8 @@ class view_take_until : public std::ranges::view_interface<view_take_until<urng_
      * \param[in] _urange The underlying range.
      * \param[in] _fun    The functor that acts as termination criterium.
      */
-    view_take_until(urng_t _urange, fun_t _fun)
-        : urange{std::move(_urange)}, fun{std::forward<fun_t>(_fun)}
+    view_take_until(urng_t && _urange, fun_t && _fun)
+        : urange{std::forward<urng_t>(_urange)}, fun{std::forward<fun_t>(_fun)}
```

Comment:
I think this requires also an adjustment of the deduction guide in Line 198. (Needs the R-References: `fun_t && _fun`)

Reply:
did you check?

Reply:
I did add the `&&`